ovos_solver_openai_persona/summarizer.py

```
@@ -0,0 +1,38 @@
+        super().__init__(config=config, translator=translator,
+                         detector=detector, priority=priority,
+                         enable_tx=enable_tx, enable_cache=enable_cache,
+                         internal_lang=internal_lang)
+        self.llm = OpenAIChatCompletionsSolver(config=config, translator=translator,
+                                               detector=detector, priority=priority,
+                                               enable_tx=enable_tx, enable_cache=enable_cache,
+                                               internal_lang=internal_lang)
+        self.prompt_template = self.config.get("prompt_template") or self.TEMPLATE
+
+
+    def get_tldr(self, document: str, lang: Optional[str] = None) -> str:
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)